synapse/handlers/typing.py

```diff
@@ -282,7 +282,7 @@ async def get_all_typing_updates(
                 typing = self._room_typing[room_id]
                 rows.append((serial, (room_id, list(typing))))
         rows.sort()
-        return rows[:limit], current_id, False
+        return rows[:limit], current_id, len(rows) > limit
```

Comment:
this is doing the wrong thing for the returned token when the limit is hit.

Reply:
I am such a crank :facepalm:

Reply:
MAYBE THIS TIME I'VE fIXeD IT?!!!??!1?